tactile_pcl/src/pcl_collector.cpp

```diff
@@ -11,9 +11,10 @@ namespace tactile {
 
 ros::Duration PCLCollector::timeout_;
 
-PCLCollector::PCLCollector(const std::string &target_frame)
+PCLCollector::PCLCollector(const std::string &target_frame, const float &threshold)
```

Comment:
The data type should be double as [`geometry_msgs::Vector3`](http://docs.ros.org/melodic/api/geometry_msgs/html/msg/Vector3.html) uses double as well.
References (or pointers) are only sensible for complex data types (class or struct instances). Basic data types (int, float, etc) should be passed as is.
Internally (under the hood) a reference is a pointer that needs to be dereferenced, which adds extra cost for basic types, but avoids (more costly) data copying for complex data types. For basic types, copying the data and copying the pointer is about the same effort.
```suggestion
PCLCollector::PCLCollector(const std::string &target_frame, const double threshold)
```